macOS/DuckDuckGo/Permissions/Model/PermissionModel.swift

```diff
@@ -81,7 +81,9 @@ final class PermissionModel {
 
     private func subscribe(to permissionManager: PermissionManagerProtocol) {
         permissionManager.permissionPublisher.sink { [weak self] value in
-            self?.permissionManager(permissionManager,
```

Comment:
This was quietly strongly referencing the `permissionManager`, but I don't know that it was really causing any issues, happy to revert this change.

Reply:
That change looks good to me, I would personally even use `[weak self, weak permissionManager]` and then  `guard let permissionManager` to make it clearer that we’re not capturing it strongly.

Reply:
Damn, thanks - good point about that syntax. Let me change it.